api/server/server.go

```diff
@@ -1160,9 +1164,11 @@ func postBuild(eng *engine.Engine, version version.Version, w http.ResponseWrite
 		authConfig        = &registry.AuthConfig{}
 		configFileEncoded = r.Header.Get("X-Registry-Config")
 		configFile        = &registry.ConfigFile{}
-		job               = eng.Job("build")
+		job               = builder.NewBuildConfig(eng.Logging, eng.Stderr)
 	)
 
+	b := &builder.BuilderJob{eng, getDaemon(eng)}
```

Comment:
Do we need this level of indirection or can we just call `Build(eng, getDaemon(eng))`  ?